source/includes/_tipser-elements.md

```diff
@@ -193,3 +193,43 @@ import { ShopComponent } from '@tipser/tipser-elements';
 
 Note: the shop component is updating the top-level page URL (when it's tabs are clicked). For this reason, please double check if it won't interfere with your web framework. 
 For the same reason, it's not recommended to include more than one `<ShopComponent />` on a single page. 
+
+
+## Style Guidelines ##
+
+Tipser Elements are the "building blocks" designed to fit your page as much as possible. We created the styling in a way that delivers a nice feel & looks from the start, but also allows you to change them easily to fit your unique sense of style. 
+
+### Collection Element ###
+
+```css
+.te-product-card {
```

Comment:
This is within section of "Collection Element". Shouldn't user actually alter its direct changes ? Or maybe the section should be called Product Element ?

Reply:
Yeah, it may be confusing as te-product-card class is used not only for collections but also for compact product view afair.

Reply:
Good point.

Reply:
@piotr-sobczyk @chmurson can you have a look if it makes more sense now?